patterns/2-structured/maturity-model.md

```diff
@@ -22,26 +19,37 @@ pain points that they are dealing with.
 
 ## Context
 
-Several teams have started adopting InnerSource practices. The exact level of
-understanding of the practice diverges between teams. The exact problems teams
-run into diverge depending on the context and working environment of each team.
-As a result the definition of what are important best practices in an
-InnerSource project differs depending on each team.
+* Several teams have started adopting InnerSource practices.
+* The exact level of understanding of the practice diverges between teams.
+* The exact problems teams run into diverge depending on the context and working environment of each team.
+* As a result the definition of what are important best practices in an InnerSource project differs depending on each team.
 
 ## Forces
 
-Teams sharing InnerSource learnings run into misunderstandings as they are not
-aware of their respective level of InnerSource adoption.
+* Teams sharing InnerSource learnings run into misunderstandings as they are not aware of their respective level of InnerSource adoption.
+* Teams believe that "it's all about migrating to a shared software development [forge](https://en.wikipedia.org/wiki/Forge_%28software%29)" (GitLab, GitHub, or Bitbucket being well known examples of such forges).
```

Comment:
Just remove this line. It's too specific.

Reply:
You mean just the line 30, right?

Interestingly the fits the discussion around what qualifies "ThinnerSource" in slack :)

Reply:
Yes, line 30 🙂